src/app/static/src/tests/components/runWorkflow/runWorkflowReport/runWorkflowReport.test.ts

```diff
@@ -542,4 +543,194 @@ describe(`runWorkflowReport`, () => {
         expect(wrapper.vm.$data.workflowRemovals).toStrictEqual(null);
         expect(wrapper.findComponent(BAlert).props("show")).toBe(false);
     });
+
+    it("can validate workflow reports", (done) => {
+        const blob = new Blob(["report"], {type: 'text/csv'});
+        blob['name'] = "test.csv";
+        const fakeFile = <File>blob;
+
+        const formData = new FormData()
+        formData.append("file", fakeFile)
+        formData.append("git_branch", "gitBranch")
+        formData.append("git_commit", "gitCommit")
+
+        const url = "http://app/workflow/validate"
+        mockAxios.onPost(url, formData).reply(200, {
+            data: [
+                {name: "minimal", params: {nmin: "5"}},
+                {name: "global", params: {p1: "v1", p2: "v2"}}
+            ]
+        });
+
+        const wrapper = getWrapper({
+            workflowMetadata: {
+                ...emptyWorkflowMetadata,
+                git_commit: "abc123",
+                git_branch: "branch"
+            }
+        });
+
+        setTimeout(async() => {
+            const fromCsvLabel = wrapper.find("#import-from-csv-label")
+
+            fromCsvLabel.find("input").trigger("click")
+
+            expect(wrapper.vm.$data.reportsOrigin).toBe("csv")
+
+            await Vue.nextTick()
+
+            expect(wrapper.find("#show-import-csv").exists()).toBe(true)
+
+            const input = wrapper.find("#show-import-csv").find("input").element as HTMLInputElement
+
+            Object.defineProperty(input, "files", {
+                value: [fakeFile]
+            })
+
+            wrapper.find("#show-import-csv").find("input").trigger("change")
+
+            setTimeout(() => {
+                expect(wrapper.vm.$data.importedFilename).toBe("test.csv")
+                expect(wrapper.vm.$data.importedFile).toMatchObject({}) // to fix incorrect assertion
+                expect(mockAxios.history.post.length).toBe(1)
+                expect(mockAxios.history.post[0].url).toBe(url)
+                expect(mockAxios.history.post[0].data).toMatchObject({})
+                done();
+            });
+        });
+    });
+
+    it("returns error if formData is empty when validating workflow", (done) => {
```

Comment:
I'm a bit confused - the logic in this test doesn't seem to match the title, and I can't see any logic in the component that would cause an error if the `formData` was empty?